### DISCUSSION TITLE
What are these arguments?

### DISCUSSION THREAD

**Discussion by Trung15010802** [rrousselGit/riverpod#3497] [Q&A] [answered]
![image](https://github.com/rrousselGit/riverpod/assets/90593296/a4411f70-49ed-4967-9f91-8f81f0d190bc)
What are these arguments? I can't find docs about them. What i can do with them?

**Comment by rrousselGit** [comment, marked as answer]
Don't use it. It's an implementation detail

If you try to use it, your IDE should complain

**Reply by rrousselGit** [reply]
This is internally used by `family`, in combination with `from`
